### PR TITLE
Updated integration titles for simplicity

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

File: docs/configuration/alertmanager-integration/grafana-alert-manager.rst [hunk 1]
```diff
@@ -3,12 +3,12 @@ Grafana AlertManager
 
 If you use Grafana alerts, you're using a `special instance of AlertManager embedded inside Grafana <https://grafana.com/docs/grafana/latest/alerting/fundamentals/alertmanager/>`_.
 
-To integrate that AlertManager with Robusta, you will need to configure both a push and a pull integration, as follows.
+You will need to configure to send alerts to Robusta.
```
Comment (aantn) [inline docs/configuration/alertmanager-integration/grafana-alert-manager.rst:6]: To configure what? I think yoiu're missing the word "Grafana"

File: docs/configuration/alertmanager-integration/grafana-alert-manager.rst [hunk 2]
```diff
@@ -3,12 +3,12 @@ Grafana AlertManager
...
 
-Configure Push Integration
+Send alerts to Robusta
 ============================
 
-A push integration sends alerts from Grafana to Robusta. To configure it:
+This integration sends alerts from Grafana to Robusta. To configure it:
```
Comment (aantn) [inline docs/configuration/alertmanager-integration/grafana-alert-manager.rst:11]: Remove unecessary words! This sentence repeats things you know from the title of the page and section.

File: docs/configuration/alertmanager-integration/_alertmanager-config.rst [hunk 3]
```diff
@@ -1,28 +1,69 @@
...
-      2. This assumes Robusta was installed in the ``default`` namespace, using a Helm release named ``robusta``.
-          * If the namespace is ``foobar``, replace ``default`` with ``foobar``
-          * If the Helm release is named ``robert`` then replace ``robusta`` with ``robert``
-      3. Keep sending alerts to receivers defined after Robusta.
-      4. Important, so Robusta knows when alerts are resolved.
+.. tab-set::
+
+    .. tab-item:: kube-prometheus-stack (Prometheus Operator)
+
+      Add the following to your `AlertManager's config Secret <https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/alerting.md#managing-alertmanager-configuration>`_
```
Comment (aantn) [inline docs/configuration/alertmanager-integration/_alertmanager-config.rst:7]: Can we update the first sentence (not here, what appears above). As you can see in the below screenshot, it's a little repetitive. ("Add the config below to the appropriate AlertManager file" and then "Add the following to your [AlertManager's config Secret")

Maybe just write "AlertManager is configured differently depending on how it was installed. Choose from the options below."

Screenshot 2024-01-01 at 13 10 38

**Review by aantn — APPROVED** [review]
Left some minor comments. Nice work.

**Comment by pavangudiwada** [comment]
@aantn Updated based on comments, take a look.

**Comment by aantn** [comment]
LGTM. Please merge.

**Comment by aantn** [comment]
Whoops, hit the wrong button. I'll go ahead and just merge ;)

**Pull request merged**